src/emmc.c

```diff
@@ -322,6 +327,20 @@ pu_emmc_write_data(PuFlash *flash,
         g_rmdir(part_mount);
     }
 
+    for (GList *c = self->clean; c != NULL; c = c->next) {
+        PuEmmcClean *clean = c->data;
+
+        if (clean->size == 0) {
+            g_warning("Size 0 specified. Skipping %lld", clean->offset);
```

Comment:
Maybe print `[...] Skipping cleaning at %lld` to let the user know we are talking about the cleaning stage here.